tools/gbench/report.py

```diff
@@ -345,6 +415,89 @@ def test_basic(self):
             self.assertEqual(len(parts), 7)
             self.assertEqual(expect_lines[i], parts)
 
+    def test_json_difference_report_output(self):
+        expected_output = [
+            {
+                'name': 'BM_SameTimes',
+                'measurements': [{'time': 0.0000, 'cpu': 0.0000, 'real_time': 10, 'real_time_other': 10, 'cpu_time': 10, 'cpu_time_other': 10}],
```

Comment:
Please format the changes.

Reply:
Will do so but I'm not sure what you're thinking on particularly? I tried to format these as much as I could so that the unit tests do not become too much spread out because of the test data. Can you please give an example of how you would like it to be?

Reply:
```
$ autopep8 -ia tools/compare.py tools/gbench/report.py tools/gbench/util.py
```

Reply:
Hmm, weird, that didn't linewrap (80 col) these lines.